runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -141,7 +141,7 @@ public void invoke() {
                     throw new IllegalStateException("Orchestrator can not be selected");
                 }
 
-            } catch (NetworkException | TimeoutException | IllegalStateException e) {
+            } catch (Exception e) {
```

Comment:
I don't think swallowing all exceptions is a good idea. The retry only needs to happen on timeout and network exceptions.

Reply:
Also, if this thread is interrupted, the interrupt flag isn't restored.

Reply:
Ok, I'll bring it back in.